front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm/index.tsx

```diff
@@ -239,43 +254,67 @@ const CustomPageSettingsForm = ({
               </Box>
             )}
 
-            {advancedCustomPagesEnabled && (
-              <Box mb={fieldMarginBottom}>
-                <Box display="flex" justifyContent="flex-start">
-                  <Label>
-                    <span>{formatMessage(messages.linkedProjectsLabel)}</span>
-                    <IconTooltip
-                      ml="10px"
-                      content={formatMessage(messages.linkedProjectsTooltip)}
-                    />
-                  </Label>
-                </Box>
-                <Box mb="30px">
-                  <Tabs
-                    name="projects_filter_type"
-                    items={projectsFilterTabs}
-                    minTabWidth={120}
-                  />
-                </Box>
-                {methods.watch('projects_filter_type') === 'topics' && (
-                  <Box mb="30px">
-                    <MultipleSelect
-                      name="topic_ids"
-                      options={mapFilterEntityToOptions(topics)}
-                      label={formatMessage(messages.selectedTagsLabel)}
-                    />
-                  </Box>
-                )}
-                {methods.watch('projects_filter_type') === 'areas' && (
-                  <Box mb="20px">
-                    <Select
-                      name="area_id"
-                      options={mapFilterEntityToOptions(areas)}
-                      label={formatMessage(messages.selectedAreasLabel)}
-                    />
-                  </Box>
-                )}
-              </Box>
+            {showAdvancedCustomPages && (
+              <LinkedProjectContainer
+                display="inline-flex"
+                disabled={!isFeatureAllowed}
```

Comment:
```suggestion
                disabled={showPlanUpgradeTease}
```

Shall we couple it to this var instead? For now it's the same, but it could get a different condition in the future. Then we only need to change showPlanUpgradeTease.

Reply:
Changes made and pushed as one commit